software-engineer/en_US/04.frontend-specialist.md

```diff
@@ -0,0 +1,33 @@
+# Frontend Specialist
+
+In this interview, we are going through a live coding, in which we are going to provide an online IDE with a boilerplate in the technology of your choice (pure javascript, react, Angular or Vue) so we can implement on screen and all logic related to it, based on low fidelity mockups.
+
+It is important to highlight that we won't run any of the written code nor check what is being rendered in the browser.
+What we will do is write the code and talk about its implementation and possible evolutions.
+
+Our goal is to simulate a pair programming, close to what we do on a daily basis here at Loft.
+We are going to present a task to be worked on that we will evolve as much as we can in the time we have available.
+
+You will be requested to share your screen so we can follow what is being developed.
+During the interview, we are going to ask questions to understand your decisions and to dig deeper on some topics of interest.
+
+You are the one that is going to guide us through this pairing and always feel free to tell us about what you are thinking, as well as any doubts that you might have along the way.
+
+Remember that there are no right or wrong answers, but many different ways to solve the same problema.  
+The most important for us is that we understand the thoughs behind your decisions!
+
+## Main Evaluation Criteria
+* Communication
+* Solution Architecture and Evolution Vision
+* How the solution was driven and concerns regarding quality and scale
+* CSS e interface implementation
+* Readability
+  * Code that is easy to read and understand
+  * Proper separation of concerns
+* Critical Sense
+  * Challenge interpretation and attention to requirements
```

Comment:
Achei esquisita essa linha, mas não achei uma proposta melhor por ora 😬

Reply:
Challenge approach and attention to requirements? 
Challenge approach and making sure requirements are met?

Reply:
Que tal:
```suggestion
  * Understanding the challenge
  * Meeting the requirements
```

Reply:
Eu gosto de usar `Understanding the challenge`, mas acho importante ressaltarmos a parte de atenção aos requisitos / entrega dos requisitos. Porque muita gente passa batido por muita coisa e muitos detalhes são importante para a gente, especialmente falando de Frontend. Acha que `Understanding the challenge` já cobre isso?

Reply:
Foi o que eu tentei adicionando a segunda linha (Meeting the requirements) 😬
Mas pode seguir aí como achar melhor, acho que qualquer uma dessas vai transmitir bem a ideia

Reply:
Eu não tinha lido direito :moises:
Gostei assim 😄